.github/workflows/ci-native.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           python-version: 3.8
       - name: Install Python Packages
         run: |
-          pip install --upgrade pip gi-docgen==2021.6 meson==0.49
+          pip install --upgrade pip gi-docgen==2021.6 meson==0.62.1
```

Comment:
It would be better to use `==0.55` here, to make sure we don't accidentally introduce use of newer Meson features.

The reason why we changed libwpe to 0.55 (and not newer) is that people integrating WPE sometimes have slightly old toolchains and/or build systems, and versions newer than Meson 0.55 do not add any feature that we would need at the moment 🤔